### PR TITLE
Add icons for additional french banks

### PR DESCRIPTION
**Pull request by rthery**
Add icon for Fortuneo (French online bank)

### CODE REVIEW

**Comment by rthery** [comment]
By the way, do you prefer one PR per icon ? Or can I submit multiple commits with just one PR ?

**Comment by dkanada** [comment]
Multiple icons in a single pull request is fine :)

**Comment by rthery** [comment]
@dkanada Great! I've added another french bank then. Thank you for initiating this project, really like all these icons!

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -1569,6 +1572,9 @@
     <!-- Mein Vodafone -->
     <item component="ComponentInfo{com.appseleration.android.selfcare/com.vis.meinvodafone.view.activity.launch.VfLaunchBaseActivity}" drawable="meinvodafone" />
 
+    <!-- Mes Comptes - LCL -->
+    <item component="ComponentInfo{fr.lcl.android.customerarea/fr.lcl.android.customerarea.activities.notconnected.SplashScreenActivity}" drawable="mescomptes-lcl" />
+
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:1577]: Could you rename the drawable without a dash? I don't think Android likes dashes in resource names.

Comment (rthery) [reply app/src/main/res/xml/appfilter.xml]: Didn't know about that sorry, just pushed a fix for it, thanks!

**Comment by Donnnno** [comment]
Suggestion to remove the small text in the icon like this and overall slightly bigger (40px wide). It will make the icon less clunky and overall better.

image


Here's my edit:

[mescompteslcl.zip](https://github.com/dkanada/frost/files/4955311/mescompteslcl.zip)

**Comment by rthery** [comment]
@Donnnno Thank you for the suggestion, I've updated the icon based on your feedback. I wasn't too sure about removing the small text, since it's in their official logo, but I never really like it I'll admit, it indeed creates too much noise to my taste!

**Comment by Donnnno** [comment]
Ah nice, Small text in icons is always ugly, almost if the de people who create it don't understand what an icon is 😬 

>since it's in their official logo 

Don't worry about official logo's too much you're the one creating icons for this pack so you have a lot more freedom ;)

**Pull request merged**